### ISSUE TITLE
Does anyone has a pre-trained model?

### ISSUE THREAD

**Issue by lood339** [chuckcho/video-caffe#46]
Hi all,
     Does anyone has a pre-trained model for UCF101? My machine is painfully slow.
     Thanks.

Jimmy

**Comment by lood339** [comment]
Here is my trained model:
https://dl.dropboxusercontent.com/u/54750216/C3D_models/c3d_ucf101_iter_38000.caffemodel
Test net output #0: accuracy/top-1 = 0.472
Test net output #1: accuracy/top-5 = 0.676
Enjoy!

**Comment by chuckcho** [comment]
@lood339 thanks for your contribution. will test and include in README later

**Comment by chuckcho** [comment]
Updated README. closing.

**Comment by pelun** [comment]
When I train this model, Test net output #0: accuracy/top-1 and Test net output #1: accuracy/top-5 very low (0-0.08). 

Can you tell me what problems is?

**Comment by xuzhenqi** [comment]
I download the pre-trained in folder `examples/c3d_ucf101/`, and run test_ucf101.sh, which produces top-1 accuracy: **0.432552**. I use batchsize 20, and run 2092 iters.

**Comment by chuckcho** [comment]
@xuzhenqi 43% accuracy doesn't look too far from the reported accuracy of 47%. @pelun. you're way off.

**Comment by pelun** [comment]
@chuckcho  What should I do to resolved this problem?

**Comment by JasmineeYang** [comment]
@chuckcho @lood339

**Comment by JasmineeYang** [comment]
@xuzhenqi @sergeyk

**Comment by lood339** [comment]
Can you have a try of this link: https://www.dropbox.com/s/lwdg0r09af6b83g/c3d_ucf101_iter_38000.caffemodel?dl=0 
The previous link is expired because dropbox changed its policyt.

**Comment by JasmineeYang** [comment]
@lood339 Thanks a lot .This one can works.I have download it.

**Comment by JasmineeYang** [comment]
@lood339 Hello, I run test_ucf101.sh, which produces top-1 accuracy: 0.211515. I use batchsize 70, and run 597 iters. I don't know that why it's different from what you said

**Comment by EthanjWang** [comment]
@lood339  @JasmineeYang @chuckcho 
Hi, I download the model shared by 'lood339'. As 'Jasminee Yang' said, the accuracy is only about 0.21.
Could you please check if this is the wrong model ? Or any of you could share the model which has accuracy of 47% ?

Thanks a lot.

**Comment by lood339** [comment]
Can you have a try of this:
model:
https://www.dropbox.com/s/lwdg0r09af6b83g/c3d_ucf101_iter_38000.caffemodel?dl=0
deploy file:
https://www.dropbox.com/s/4wdaj5oiml6avqw/c3d_ucf101_deploy.prototxt?dl=0

Jimmy

**Comment by EthanjWang** [comment]
@lood339 Hi Jimmy,
I am very sorry to tell you that this model is exactly the same as the one you shared before, which got an accuracy around 21%.
If possible, would you please double check for the correct one.
Thanks in advance.
Ethan

**Comment by junmuzi** [comment]
@lood339 Hi Jimmy,
Could you send me a c3d_ucf101_deploy.prototxt? I can not visit(https://www.dropbox.com/s/4wdaj5oiml6avqw/c3d_ucf101_deploy.prototxt?dl=0), my email is junmuzi@126.com, Thanks very much.

**Comment by junmuzi** [comment]
@EthanjWang , Hi Ethan,
Could you help to send me a c3d_ucf101_deploy.prototxt? I want to test the c3d_ucf101_iter_38000.caffemodel, too. 
Thanks a lot.
Jun

**Comment by junmuzi** [comment]
@lood339 @EthanjWang , Hi,
I also test above model using the corresponding deploy file, but hit a serious issue.
The output of check_data_and_model.py is the same all the time using model "./c3d_ucf101_iter_38000.caffemodel" when I input different images(I have tried fish-bike.jpg and cat.jpg). I am so surprised and puzzled. Maybe I have got some mistakes anywhere, could anyone give me some suggestion? Thanks.

The follow is my debug info(based on 25bc6d):

(Pdb) p output_prob
array([ 0.00580123,  0.00344231,  0.00512505,  0.02758851,  0.02166136,
        0.0047916 ,  0.00347026,  0.02464976,  0.00655003,  0.00455923,
        0.01620403,  0.00231314,  0.00538909,  0.01927903,  0.00329624,
        0.00759618,  0.00291226,  0.00790412,  0.00663601,  0.00576234,
        0.00265791,  0.06774281,  0.00394197,  0.00217366,  0.0103282 ,
        0.01745262,  0.00499696,  0.00596806,  0.01204872,  0.01465816,
        0.00390601,  0.0154008 ,  0.00425976,  0.00915863,  0.01411434,
        0.00991541,  0.00278897,  0.01102778,  0.00692579,  0.01100748,
        0.00544623,  0.01335799,  0.00461721,  0.00599697,  0.00970814,
        0.00304217,  0.00306839,  0.00074699,  0.02649253,  0.00471338,
        0.00893008,  0.00684513,  0.00769747,  0.01527166,  0.00784316,
        0.00401985,  0.00626187,  0.01174736,  0.00355275,  0.00587889,
        0.00333939,  0.00414828,  0.00603733,  0.00444678,  0.00319782,
        0.00324162,  0.00606149,  0.0392826 ,  0.0051594 ,  0.00588619,
        0.00756048,  0.01787155,  0.00951777,  0.02192619,  0.02281141,
        0.00852187,  0.0024029 ,  0.00562544,  0.00997402,  0.02348322,
        0.02809488,  0.01101367,  0.01037497,  0.00545539,  0.00268822,
        0.01033187,  0.00794003,  0.00903882,  0.00927738,  0.00494286,
        0.00861302,  0.00618246,  0.01118466,  0.0049173 ,  0.00323196,
        0.00650915,  0.01022164,  0.04497042,  0.00473391,  0.01566175,
        0.00347585], dtype=float32)
(Pdb) p output_prob.argmax()
21

**Comment by JasmineeYang** [comment]
Hello，Jimmy.Do you still have the file of train.log ?I don't know why my train.log is empty while my test.log is exist.If you do,could you send me it?

At 2017-04-19 00:30:34, "Jimmy" <notifications@github.com> wrote:
Can you have a try of this:
model:
https://www.dropbox.com/s/lwdg0r09af6b83g/c3d_ucf101_iter_38000.caffemodel?dl=0
deploy file:
https://www.dropbox.com/s/4wdaj5oiml6avqw/c3d_ucf101_deploy.prototxt?dl=0

Jimmy
